<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>쓰레기 분리수거 프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="submit_Button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>그래프 그리기</string>
    </property>
   </widget>
   <widget class="QLabel" name="chart_container">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>391</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_Button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_text_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>421</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
